### PR TITLE
Slurp: leading 0 padding on Mondo IDs

### PR DESCRIPTION
**Pull request by joeflack4**
Updates
```
Slurp
- Added leading 0 padding to Mondo IDs
```

### CODE REVIEW

File: src/scripts/migrate.py [hunk 1]
```diff
@@ -75,8 +75,7 @@ def slurp(
                 break
         if qualified_parents:
             next_mondo_id, mondo_term_ids = _get_next_available_mondo_id(next_mondo_id, max_id, mondo_term_ids)
-            # mondo_id = 'MONDO:' + ('0' * (len(str(next_mondo_id)) - 7)) + str(next_mondo_id)  # min->max needs no pad
-            mondo_id = 'MONDO:' + str(next_mondo_id)
+            mondo_id = 'MONDO:' + ('0' * (len(str(next_mondo_id)) - 7)) + str(next_mondo_id)  # min->max needs no pad
```
Comment (joeflack4) [inline src/scripts/migrate.py:78]: The padding

Comment (matentzn) [reply src/scripts/migrate.py:78]: I  think it is more robust to use zfill:

https://stackoverflow.com/questions/39402795/how-to-pad-a-string-with-leading-zeros-in-python-3

**Review by matentzn — CHANGES_REQUESTED** [review]

File: src/scripts/migrate.py (hunk 1 above)
Comment (joeflack4) [reply src/scripts/migrate.py:78]: Ok; didn't even know that existed. This is such a simple programming problem that I didn't even think twice. I'll switch to use that, though.

Comment (joeflack4) [reply src/scripts/migrate.py]: done

**Review by matentzn — APPROVED** [review]

**Pull request merged**